sim/pokemon.ts

```diff
@@ -330,8 +330,10 @@ export class Pokemon {
 		set.level = this.battle.clampIntRange(set.adjustLevel || set.level || 100, 1, 9999);
 		this.level = set.level;
 		const genders: { [key: string]: GenderName } = { M: 'M', F: 'F', N: 'N' };
-		this.gender = genders[set.gender] || this.species.gender || (this.battle.random(2) ? 'F' : 'M');
-		if (this.gender === 'N') this.gender = '';
+		this.gender = '';
+		if (this.battle.gen < 6 || this.battle.ruleTable.has('obtainablemisc')) {
+			genders[set.gender] || this.species.gender || (this.battle.random(2) ? 'F' : 'M');
```

Comment:
```suggestion
			this.gender = genders[set.gender] || this.species.gender || (this.battle.random(2) ? 'F' : 'M');
```
something tells me you forgot that part